subworkflows/local/standardisation_profiles.nf

```diff
@@ -22,20 +48,33 @@ workflow STANDARDISATION_PROFILES {
     ch_multiqc_files       = Channel.empty()
 
     //Taxpasta standardisation
-    ch_input_for_taxpasta = profiles
+    ch_prepare_for_taxpasta = profiles
                             .map {
                                     meta, profile ->
                                         def meta_new = [:]
                                         meta_new.id = meta.db_name
-                                        meta_new.tool = meta.tool == 'metaphlan3' ? 'metaphlan' : meta.tool == 'malt' ? 'megan6' : meta.tool
+                                        meta_new.tool = meta.tool == 'metaphlan' ? 'metaphlan' : meta.tool == 'malt' ? 'megan6' : meta.tool
```

Comment:
I think you only need to check for `malt`

Reply:
Now yes, agreed! I think we indeed didn't make it backwards compatible

Reply:
Done!